examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java

```diff
@@ -64,7 +64,7 @@ public KubernetesSeedProvider(Map<String, String> params) {
 
         String host = protocol + "://" + hostName + ":" + hostPort;
         String serviceName = getEnvOrDefault("CASSANDRA_SERVICE", "cassandra");
-        String path = "/api/v1beta1/endpoints/";
+        String path = "/api/v1beta3/endpoints/";
```

Comment:
The only other URL I found was in rethinkdb, which hasn't been converted yet.

Good catch, thanks.